WordPress/src/androidTest/java/org/wordpress/android/e2e/BlockEditorTests.kt

```diff
@@ -72,4 +72,42 @@ class BlockEditorTests : BaseTest() {
             .switchToVisualMode()
             .verifyPostElementText(mPostText)
     }
+
+    @Test
+    fun e2eBlockEditorCanUndoChanges() {
+        val title = "blockEditorCanUndoChanges"
+        MySitesPage()
+            .startNewPost()
+        BlockEditorPage()
+            .waitForTitleDisplayed()
+            .enterTitle(title)
+            .enterParagraphText(mPostText)
+            .verifyContentStructure(1, mPostText.split(" ").count(), mPostText.length)
+            .dismissContentStructure()
+            .undo()
+            .undo()
+            .verifyContentStructure(0, 0, 0)
+            .dismissContentStructure()
+    }
+
+    @Test
+    fun e2eBlockEditorCanRedoChanges() {
+        val title = "blockEditorCanRedoChanges"
+        MySitesPage()
+            .startNewPost()
+        BlockEditorPage()
+            .waitForTitleDisplayed()
+            .enterTitle(title)
+            .enterParagraphText(mPostText)
+            .verifyContentStructure(1, mPostText.split(" ").count(), mPostText.length)
+            .dismissContentStructure()
+            .undo()
+            .undo()
+            .verifyContentStructure(0, 0, 0)
+            .dismissContentStructure()
+            .redo()
+            .redo()
+            .verifyContentStructure(1, mPostText.split(" ").count(), mPostText.length)
+            .dismissContentStructure()
+    }
```

Comment:
`e2eBlockEditorCanUndoChanges()` is actually part of `e2eBlockEditorCanRedoChanges()`. We can avoid duplication by keeping only the later and maybe renaming it to something like `e2eBlockEditorCanUndoRedoChanges()`?

Reply:
Good idea! I've updated it to unify them with the suggested naming https://github.com/wordpress-mobile/WordPress-Android/pull/18705/commits/86a3db82b4c0b86782e9d83cd2e838cdc8d713e7